packages/workflow/src/interceptors.ts

```diff
@@ -80,6 +80,7 @@ export interface ActivityInput {
   readonly options: ActivityOptions;
   readonly headers: Headers;
   readonly seq: number;
+  readonly cmdOpts?: WorkflowCommandOptions;
```

Comment:
I don't think we should have this `WorkflowCommandOptions` type. I even wonder if we should inline `summary` and `details` directly in `ActivityInput` (without the extra `UserMetadata` nesting). Or add these two to `ActivityOptions`, though that might pose some minor challenge regarding the type provided to the `proxyActivities()` function.

Reply:
What did we do in other SDKs?